next/src/types.ts

```diff
@@ -76,7 +76,7 @@ export type JsfSchema = JSONSchema & {
   // Extra validations to run. References validations in the `x-jsf-logic` root property.
   'x-jsf-logic-validations'?: string[]
   // Extra attributes to add to the schema. References computedValues in the `x-jsf-logic` root property.
-  'x-jsf-logic-computedAttrs'?: Partial<Record<keyof NonBooleanJsfSchema, string | JsfSchema['x-jsf-errorMessage']>>
+  'x-jsf-logic-computedAttrs'?: Record<string, string | object>
```

Comment:
These types were really not adding much here so I decided to go back to a simpler one